regression/invariants/Makefile

```diff
@@ -0,0 +1,38 @@
+
+default: tests.log
+
+SRC = driver.cpp
+
+INCLUDES = -I ../../src
+
+OBJ += ../../src/util/util$(LIBEXT)
+
+include ../../src/config.inc
+include ../../src/common
+
+test: driver$(EXEEXT)
+	@if ! ../test.pl -c ../../../src/cbmc/cbmc ; then \
```

Comment:
How does this work? Don't you need some shell script that compiles and runs `driver.cpp` rather than running `cbmc`?